Apromore-Custom-Plugins/Access-Control-Portal-Plugin/src/main/java/org/apromore/plugin/portal/accesscontrol/controllers/AccessController.java

```diff
@@ -498,15 +503,20 @@ private String formatPath(FolderType folder, String name) {
    * Apply the changes in the access control by comparing assignment listbox with previous access
    * control list
    */
-  private void applyChanges() {
+  private boolean applyChanges() {
     Map<Group, AccessType> groupAccessTypeChanges = new HashMap<>(groupAccessTypeMap);
 
     for (Assignment assignment : assignmentModel) {
       String rowGuid = assignment.getRowGuid();
       boolean shareUserMetadata;
       shareUserMetadata = false;
       AccessType accessType = AccessType.getAccessType(assignment.getAccess());
+
       Group group = securityService.findGroupByRowGuid(rowGuid);
+      if (!isValidOwnerAssignment(assignment,group)){
+        Notification.error(getLabel("admin_owner_error_message"));
+        return false;
+      }
```

Comment:
We should still be able to share files with admins/superusers and give them Owner access.